### PR TITLE
℮: change nuclear disk to ℮

### PR DESCRIPTION
**Pull request by ficcialfaint**
# Lmk if the copyright is wrong cuz I have no idea what it's supposed to be since this is just a Unicode symbol

## About the PR

Nuclear authentication disk is now ℮

## Why / Balance

＼（〇_ｏ）／

## Media

e

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

### CODE REVIEW

File: Resources/Textures/Objects/Misc/nukedisk.rsi/meta.json [hunk 1]
```diff
@@ -1,20 +1,14 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/32a2e3e4ceda3ef442c1496cbc6c6a896d684702",
+  "copyright": "Taken from a Unicode Character “℮” Estimated Symbol (U+212E)",
```
Comment (0x6273) [inline Resources/Textures/Objects/Misc/nukedisk.rsi/meta.json:4]: This should mention what font it's from and license should be updated accordingly.

**Comment by mirrorcult** [comment]
℮..

**Comment by lzk228** [comment]
explain the joke please

**Comment by deltanedas** [comment]
erlang ss13 rewrite

**Comment by FairlySadPanda** [comment]
e

**Comment by Simyon264** [comment]
what exactly is the joke? I don't really get it. 

℮

**Comment by Hiucko** [comment]
**### _e_**?

**Comment by ficcialfaint** [comment]
> what exactly is the joke? I don't really get it.

> explain the joke please

That's not a joke, more like an easter egg to an album "E" made by Ecco2k that uses ℮ as an album cover

**Comment by lzk228** [comment]
then big ass nuclear disc #26131 is more funny than this

File: Resources/Textures/Objects/Misc/nukedisk.rsi/meta.json (hunk 1 above)
Comment (ficcialfaint) [reply Resources/Textures/Objects/Misc/nukedisk.rsi/meta.json]: What should be put in `license` then? The font is licensed under Apache 2.0, but if I put it into the `license` field, RSI Validator says it's not one of the CC-* ones

Comment (0x6273) [reply Resources/Textures/Objects/Misc/nukedisk.rsi/meta.json]: "Custom" and mention that the font is Apache 2.0 in copyright field.

Comment (ficcialfaint) [reply Resources/Textures/Objects/Misc/nukedisk.rsi/meta.json]: Custom doesn't seem to work either

**Comment by VasilisThePikachu** [comment]
Don't really think this is funny imo. Plus the licensing stuff is a little annoying. Thank you for your pr

**Pull request closed without merging**